var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -387,6 +387,11 @@ def cmake_args(self):
                 '-DMPI_CXX_COMPILER=%s' % spec['mpi'].mpicxx,
                 '-DMPI_Fortran_COMPILER=%s' % spec['mpi'].mpifc,
             ])
+            if '+cuda' in spec:
+                options.extend([
+                    '-DDEAL_II_MPI_WITH_CUDA_SUPPORT:BOOL=ON',
+                    '-DCUDA_HOST_COMPILER=%s' % spec['mpi'].mpicxx
```

Comment:
I could be mistaken, but I think that this should be `CMAKE_CUDA_HOST_COMPILER`? See https://gitlab.kitware.com/cmake/cmake/-/issues/19013

Reply:
Ah, I now see in the documentation that you [mentioned](https://cmake.org/cmake/help/v3.17/module/FindCUDA.html) that `CUDA_HOST_COMPILER` is listed. I wonder which one is indeed correct?

Reply:
I used cmake 3.17, and CUDA_HOST_COMPILER was picked up by more parts of cmake, but still didn't completely work - see discussion below.